src/content.ts

```diff
@@ -971,8 +967,7 @@ function getYouTubeVideoIDFromURL(url: string): string | boolean {
         return id.length == 11 ? id : false;
     } else if (urlObject.pathname.startsWith("/embed/") || urlObject.pathname.startsWith("/shorts/")) {
         try {
-            const id = urlObject.pathname.split("/")[2];
-            if (id && id.length >= 11) return id.substr(0, 11);
+            return urlObject.pathname.split("/")[2].slice(0, 11);
```

Comment:
Why this change?

Reply:
slice won't return a different value if it's already under the length, adding the check for id and id.length isn't necessary

Reply:
ah I see what you mean

Reply:
reverted